libs/vkd3d/utils.c

```diff
@@ -635,6 +635,137 @@ HRESULT return_interface(void *iface, REFIID iface_iid,
     return hr;
 }
 
+const char* debug_dxgi_format(DXGI_FORMAT format)
```

Comment:
The style is `const char *foo` not `const char* foo`.

Reply:
Resolved locally.